randovania/games/am2r/pickup_database/progressive_items.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import annotations
+
+
+def tuples() -> tuple:
```

Comment:
This should be a list of tuples, where the first element in the list is a string (the progressive name) and the second element is a tuple with an arbitrary amount of strings for the non-progressive name.
So `list[tuple[str, tuple[str, ...]]]` ?

Reply:
Would be great to define a type alias somewhere for this mess haha

Reply:
Yeah, as all the other games will need the same type hint. Should i do that in this PR, or just add a TODO for the future?

Reply:
Hmmm in feature freeze I wouldn't change anything in other games but it's also just a type hint.
I would say: Add a type alias somewhere where other games can use it in the future, which means don't just put in the AM2R specific file (I hope that there exists a nice spot for it)